index.html

```diff
@@ -3,12 +3,13 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="../css/header.css">
-    <link rel="stylesheet" href="../css/sidebar.css">
-    <link rel="stylesheet" href="../css/footer.css">
-    <link rel="stylesheet" href="../css/general.css">
+    <link rel="stylesheet" href="./css/header.css">
+    <link rel="stylesheet" href="./css/sidebar.css">
+    <link rel="stylesheet" href="./css/footer.css">
+    <link rel="stylesheet" href="./css/general.css">
     <title>Home</title>
 </head>
 <body>
+    <footer> Copyright &copy; 2024 Zoo Topia || AADAM Project || E-mails </footer>
 </body>
```

Comment:
You should probably add some more stuff to the footer.
Instead of dividing the different paragraphs with two lines, use divs and organize them using a flex row.